src/utils/stopwatch.ts

```diff
@@ -1,19 +1,28 @@
 class Stopwatch {
 
-    private start: number = 0;
-    private end: number = 0;
+    private _start: number = 0;
+    private _end: number = 0;
 
     startWatch() {
-        this.start = new Date().getMilliseconds();
+        this._start = Date.now();
     }
 
     stopWatch() {
-        this.end = new Date().getMilliseconds();
+        this._end = Date.now();
     }
 
     getDuration(): number {
-        return this.end - this.start;
+        return this._end - this._start;
     }
+
+    get start(): number {
+        return this._start;
+    }
+
+    get end(): number {
+        return this._end;
+    }
+
```

Comment:
Following up the previous comment, please remove this. We shouldn't expose the internal fields unless we have a clear use case for this.

Reply:
Isnt there a flaw in code?

const totalDurationStopWatch = new Stopwatch();
        totalDurationStopWatch.startWatch();
startWatch() method assigns new value to _start var in the Stopwatch class.
Then the variable's value is overwritten by the second call inside the do while loop:
do {
            const stopWatch = new Stopwatch();
            stopWatch.startWatch();
That is why I have decided to capture the value of the _start variable before the second call of the method inside the loop.
How can I get this value then instead of using the getter method?

Reply:
I will give you a hint here: If you want to put code in comments, use a code block :)

Reply:
Regarding the new issue of reassigning `_start` field. Yes, it can be an issue. Please unit test it to prove your point and if needed open a new issue once this one is fixed.

Reply:
> How can I get this value then instead of using the getter method?

Make a new `StopWatch`. 😃